packages/ui/src/checkbox/CheckboxField.stories.tsx

```diff
@@ -52,10 +52,20 @@ export const labelPlacement = () => (
 export const inlineForm = () => (
   <CheckboxGroupField FormGroupProps={{ row: true }}>
     <FormGroup row={true}>
-      <RadioField label="Radio" />
-      <CheckboxField label="Checkbox" />
-      <FormControlLabel label="Switch" control={<Switch />} />
-      <TextField placeholder="Text Field" />
+      <Columns align="center" space="small">
+        <Column width="content">
```

Comment:
@umrzoq-toshkentov Do we use `FormGroup` somewhere in the codebase? If so, can we change the spaces for `FormGroup` instead of using `Columns`?

Reply:
@timurhamitov yes `FormGroup` is used Carrier TMS. There is just updated Columns in case Artyom asked more space in example that is why I have just updated. There is not a big side effect here.

Reply:
@umrzoq-toshkentov Can you ask Artyom did he meant just for examples or for FormGroup? You can show him forms in Carrier TMS that use FormGroup.
But I think we also use Columns sometimes for inline some fields. So it will not fix all cases.
But we need to be on the same page with the design team.

Reply:
@timurhamitov yes I have discussed with him and everything is fine